mmv1/third_party/terraform/services/compute/resource_compute_storage_pool_test.go

```diff
@@ -0,0 +1,74 @@
+package compute_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/plancheck"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+)
+
+func TestAccComputeStoragePool_computeStoragePool_update(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckComputeStoragePoolDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccComputeStoragePool_computeStoragePoolFullExample(context),
+			},
+			{
+				ResourceName:            "google_compute_storage_pool.test-storage-pool-full",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"deletion_protection", "labels", "terraform_labels", "zone"},
+			},
+			{
+				Config: testAccComputeStoragePool_computeStoragePool_update(context),
+				ConfigPlanChecks: resource.ConfigPlanChecks{
+					PreApply: []plancheck.PlanCheck{
+						plancheck.ExpectResourceAction("google_compute_storage_pool.test-storage-pool-full", plancheck.ResourceActionUpdate),
+					},
+				},
+			},
+			{
+				ResourceName:            "google_compute_storage_pool.test-storage-pool-full",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"deletion_protection", "labels", "terraform_labels", "zone"},
+			},
+		},
+	})
+}
+
+func testAccComputeStoragePool_computeStoragePool_update(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+resource "google_compute_storage_pool" "test-storage-pool-full" {
+  name = "tf-test-storage-pool-full%{random_suffix}"
+
+  description = "Hyperdisk Balanced storage pool"
+
+  capacity_provisioning_type   = "STANDARD"
+  pool_provisioned_capacity_gb = "11264"
```

Comment:
Can we lower the capacity on the tests tests. Looks like we are exceeding quota on out projects

Reply:
Hyperdisk storage pools requires minimal initial capacity 10 TiB + 1 TiB increment. The test capacity is smallest size. 

```
Minimum storage pool provisioned capacity	10 TiB
Storage capacity increments	multiples of 1 TiB
```
https://cloud.google.com/compute/docs/disks/storage-pools#hdbsp-limits

Reply:
Okay I put a quota request in. As is these few tests will outstrip current quota